protocol/contracts/fakes/v2/strategies/DsrBaseStrategyTestnet.sol

```diff
@@ -184,22 +184,21 @@ contract DsrBaseStrategyTestnet is AbstractStrategy, ITempleBaseStrategy {
     /**
      * @notice The TRV is able to withdraw on demand in order to fund other strategies which 
      * wish to borrow from the TRV.
-     * @dev It may withdraw less than requested if there isn't enough balance in the DSR.
+     * @dev This will revert if the requestedAmount is more than the balance in the DSR.
```

Comment:
The original intent was that TRV might try and pull funds out of the base strategy, and the base strategy may not have 100% of what was requested.

But actually that just adds complexity, and really the TRV logic will only try and withdraw an amount from the base strategy for a valid amount. If the funds aren't there, it should simply revert. This simplifies the logic since it doesn't have to cater for when the amount received is less than what was requested.